pom.xml

```diff
@@ -274,6 +274,11 @@
         <version>${project.version}</version>
         <scope>test</scope>
       </dependency>
+    <dependency>
+        <groupId>com.esotericsoftware</groupId>
```

Comment:
Also indented a little too far. Is updating the kryo version part of this change too?